scene/gui/tab_container.cpp

```diff
@@ -789,6 +789,11 @@ Control *TabContainer::get_current_tab_control() const {
 void TabContainer::remove_child_notify(Node *p_child) {
 	Container::remove_child_notify(p_child);
 
+	Control *c = Object::cast_to<Control>(p_child);
+	if (!c) {
```

Comment:
Maybe this since we don't need `c` further down:
```suggestion
	if (!Object::cast_to<Control>(p_child)) {
```